### PR TITLE
Merge frontend changes

### PR DESCRIPTION
**Pull request by SteveFromAccounting**
Just some minor ones

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,3 +30,36 @@ Running during normal development:
...
+######  If you have chosen to install both Vagrant and VirtualBox
+Run the following commands in a terminal to get started:
+```sh
+
+$ cd <code dir>/team9sofine
+$ npm install
+$ gulp build
+```
+
+Now, in your browser, go to `http://192.168.33.110` and you should see the frontend loaded. Within the terminal, you can also run `$lynx myapp.dev` to see a text-only version. Ctrl+C to quit out.
```
Comment (dandoman) [inline README.md:51]: Should this be 'localhost' or 127.0.0.1 or ::1 ? If not, I'm missing something unless this is one of those magic addresses

File: static-sample/js/create-load.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+$(document).ready(function() {
```
Comment (dandoman) [inline static-sample/js/create-load.js:1]: This looks like my ghetto js. No need to keep it. Unless you're using it as some kind of base to build on, then you should pull master and rebase to get the latest.

File: static-sample/js/create-kink-load.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
+$(document).ready(function() {
+				row = $("<tr></tr>");
```
Comment (dandoman) [inline static-sample/js/create-kink-load.js:2]: +1 ghetto js removal.

**Comment by SteveFromAccounting** [comment]
Oh, you can squash merge this if you want into just 'frontend changes', so it's not as messy. Just keep the hyphenated pieces.

File: README.md (hunk 1 above)
Comment (SteveFromAccounting) [inline README.md:51]: No, it's the address to access the virtual machine that Vagrant creates.

File: static-sample/js/create-load.js (hunk 2 above)
Comment (SteveFromAccounting) [inline static-sample/js/create-load.js:1]: It's there temporarily so we don't lose your work while I build up the framework to do it in a more modern way. Unless you mean that this code isn't used?

File: README.md (hunk 1 above)
Comment (SteveFromAccounting) [inline README.md:51]: See https://github.com/dandoman/team9sofine/blob/master/Vagrantfile#L5

File: static-sample/js/create-kink-load.js (hunk 3 above)
Comment (SteveFromAccounting) [inline static-sample/js/create-kink-load.js:2]: oh crap, I see what you mean now. I forgot to `git rm -r` that stuff during the rebase. Forget this PR then. I'll rebase, squash the commits, and make it all a little cleaner.

**Pull request merged**